recipes/full_finetune_distributed.py

```diff
@@ -203,6 +210,7 @@ def _setup_model(
         cfg_model: DictConfig,
         enable_activation_checkpointing: bool,
         model_state_dict: Dict[str, Any],
+        mode_tie: bool = False,
```

Comment:
```suggestion
        model_tie: bool = False,
```

Reply:
Thank you for your suggestion, I have fixed it in the latest version.